### DISCUSSION TITLE
Tutorial for ppi

### DISCUSSION THREAD

**Discussion by mvfferraz** [DeepRank/deeprank2#566] [Q&A] [answered]
Hey all,

First of all congratulations on this amazing work. I don't know if I'm being too early here as this is the first topic on discussion, but I have had some problems to reproduce the tutorial for PPI.

I have installed deeprank2 locally and everything seems to run fine. I'm running the JN for "data_generation_ppi.ipynb".
All the command works except for queries processing. I have the following error:

"Graph/Query with ID residue-ppi:M-P:BA-105857 ran into an Exception (ValueError: chain ['M', 'P'] not found in the structure), and it has not been written to the hdf5 file. More details below:
chain ['M', 'P'] not found in the structure
Traceback (most recent call last):
  File "/home/matheus/miniconda3/envs/deeprank2/lib/python3.10/site-packages/deeprank2/query.py", line 477, in _process_one_query
    graph = query.build(self._feature_modules)
  File "/home/matheus/miniconda3/envs/deeprank2/lib/python3.10/site-packages/deeprank2/query.py", line 202, in build
    graph = self._build_helper()
  File "/home/matheus/miniconda3/envs/deeprank2/lib/python3.10/site-packages/deeprank2/query.py", line 364, in _build_helper
    contact_atoms = get_contact_atoms(
  File "/home/matheus/miniconda3/envs/deeprank2/lib/python3.10/site-packages/deeprank2/utils/buildgraph.py", line 176, in get_contact_atoms
    atom_indexes = interface.get_contact_atoms(cutoff=influence_radius, chain1=chain_id1, chain2=chain_id2)
  File "/home/matheus/miniconda3/envs/deeprank2/lib/python3.10/site-packages/pdb2sql/interface.py", line 82, in get_contact_atoms
    raise ValueError(
**ValueError: chain ['M', 'P'] not found in the structure**
The queries processing is done. The generated HDF5 files are in data_processed/ppi/residue.
"

However, I'm using the input provided in the tutorial for peptide-MHC systems, in which the chains are indeed M and P.

Has anyone faced this problem before? I'd really appreciate thoughts on this matter.

Best,
Matheus

**Comment by gcroci2** [comment, marked as answer]
Hi @mvfferraz, sorry for the delay, somehow I missed your post.

This seems more like a topic for a new [issue](https://github.com/DeepRank/deeprank2/issues) (which may be an installation error or a bug) than for the Discussions tab. Please fill out this form [here](https://github.com/DeepRank/deeprank2/issues/new?assignees=&labels=bug&projects=&template=bug_report.md&title=Bug%3A+%3Cplease+fill+here%3E), we will pick it up asap.

**Reply by mvfferraz** [reply]
Dear Giulia, thank you for the reply.

I've managed to fix this issue, but I still have some problems trying to reproduce the GNN training tutorial. I'll fill out the form in the proper venue.

Thank you and have a nice day.
